odata_query/sqlalchemy/shorthand.py

```diff
@@ -1,31 +1,66 @@
-from typing import List
```

Comment:
The imports aren't organized correctly. We use `isort` to automate this process, as described in https://github.com/gorilla-co/odata-query/blob/master/CONTRIBUTING.rst#local-development. I did notice that our Github Workflow wasn't configured to run on Pull Requests, otherwise this would've been more clear, sorry! This is improved for future PRs.

Reply:
My bad. I did run `blake` but forgot to run `isort`. I had some issues with poetry, so I wasn't able to properly test the PR. Please do not hesitate to make any changes you think are necessary, or let me know if it is something I can do.